### PR TITLE
ACCUMULO-4867: Java Collections Improvements in DefaultLoadBalancer

### PR DESCRIPTION
**Pull request by belugabehr**

### CODE REVIEW

File: server/base/src/main/java/org/apache/accumulo/server/master/balancer/DefaultLoadBalancer.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
 import java.util.Objects;
 import java.util.Set;
 import java.util.SortedMap;
+import java.util.concurrent.ThreadLocalRandom;
```
Comment (ctubbsii) [inline server/base/src/main/java/org/apache/accumulo/server/master/balancer/DefaultLoadBalancer.java]: Don't need this import.

**Comment by busbey** [comment]
Is this waiting on something for merge?

**Comment by ctubbsii** [comment]
I was going to merge it after I do a quick build test to ensure no formatting fixes are required and that it doesn't introduce any new build warnings. I haven't had time, but I can do so now.

**Comment by ctubbsii** [comment]
Hi @belugabehr, thanks for contributing. This is your second PR merged. If you're interested in contributing regularly, please consider subscribing to the [developer mailing list][1] and introducing yourself, and possibly describing which areas of the Accumulo project you're most interested in helping out with, so we can welcome you to our community and get to know you better.

[1]: https://accumulo.apache.org/contact-us/

**Pull request merged**